movement/io/load_bboxes.py

```diff
@@ -320,13 +324,15 @@ def from_via_tracks_file(
     logger.debug(f"Validated VIA tracks .csv file {via_file.path}.")
 
     # Create an xarray.Dataset from the data
-    bboxes_arrays = _numpy_arrays_from_via_tracks_file(via_file.path)
+    bboxes_arrays = _numpy_arrays_from_via_tracks_file(
+        via_file.path, via_file.frame_regexp
```

Comment:
I am bit confused by one fact. You have gone to lengths to generalise the frame regex by having it supplied as a parameter to the private funcs, but it is not exposed in the public functions, which means you don't expect the user to change it. 

What was your motivation for generalising th regex, as opposed to hard-coding it? Is it maintainability and robustness to future file format changes?

Reply:
PS: I'm not opposing the choice, just being curious.

Reply:
yes, I think it would be good to expose the regexp actually.

I added that to the PR and some more tests. Will re-request review if that is ok (hopefully it's quick)

thanks!